mutiny-core/src/lib.rs

```diff
@@ -776,6 +779,22 @@ impl<S: MutinyStorage> MutinyWallet<S> {
         self.node_manager
             .pay_invoice(None, inv, amt_sats, labels)
```

Comment:
should we just check the node manager balance first and see if we could even pay it?

Reply:
seems like an unnecessary call that'll happen anyways and produce the same exact error

Reply:
Decided it was an okay idea to go ahead and do. It'll do the call mostly twice but whatever.